examples/spheres.py

```diff
@@ -0,0 +1,21 @@
+"""
+Display two spheres with Surface layers
+"""
+
+import napari
+from meshzoo import icosa_sphere
```

Comment:
If it is only used in this example, I would lean towards the try/except.

Reply:
I used the same library in #3252's example; should we put this block in both or just add this to the deps? I lean toward adding to the deps, since it's a super convenient library for quick mesh generation and we might use it elsewhere in the examples (and maybe tests with screenshots down the line).

Reply:
> EDIT: actually, I already added to the deps in that PR a couple of days ago. Unless someone disagrees, I'll go forward with that!

Looks like it's just getting added to the test dependencies there, so I'd still add the try/ except block just in case someone doesn't have it installed.

Looks like the package itself has relatively light dependencies - only numpy, https://github.com/nschloe/meshzoo, so you could add a `napari_provide_sample_data` option for it like we did for scikit-image here https://github.com/napari/napari/blob/559eb525194267bb4ebb38e2987006f8883f142e/napari/plugins/_skimage_data.py would definitely make it a lot easier for people to then use the meshdata right from the drop-down menu. Thoughts everyone?

Reply:
Ahn I see! Ok, will do. Yeah, that sounds useful!

Reply:
if you add a `provide_sample_data` then i think we should add `meshzoo` to the `[all]` dependencies too here https://github.com/napari/napari/blob/84362f9ea3064a850cd7227f689c8829fd85170f/setup.cfg#L88-L89, what do @tlambert03 and @jni think about it? it seems generally useful with little cost to me, but i understand wanting to keep dependencies light too

Reply:
Isn't this a good use case for optional dependencies, and maybe a popup when `provide_sample_data` is called that prompts the user to install `meshzoo`?

Reply:
no popups!

@sofroniewn I actually think we should add a `docs` dependency list that includes everything needed for our examples and documentation. Then `all` should reference that also, yes. All should mean all. 😂

Reply:
I've created #3289 to track this